pkg/security/events/rate_limiter.go

```diff
@@ -27,7 +27,7 @@ var (
 	defaultLimit = rate.Limit(10)
 	// Default Token bucket size. 40 is meant to handle sudden burst of events while making sure that we prevent
 	// flooding.
-	defaultBurst int = 40
+	defaultBurst = 40
```

Comment:
nit: i think this should be a const (since this whole thing is kinda like a const block but with non-const-able types)

Reply:
good point, fixed in last commit